packages/react-native/ReactCommon/react/nativemodule/core/platform/android/ReactCommon/JavaTurboModule.cpp

```diff
@@ -57,16 +57,32 @@ JavaTurboModule::~JavaTurboModule() {
 
 namespace {
 
+constexpr static auto kReactFeatureFlagsJavaDescriptor = "com/facebook/react/config/ReactFeatureFlags";
+
+static bool getFeatureFlagBoolValue(const char *name) {
+    static const auto reactFeatureFlagsClass = facebook::jni::findClassStatic(kReactFeatureFlagsJavaDescriptor);
+    const auto field = reactFeatureFlagsClass->getStaticField<jboolean>(name);
+    return reactFeatureFlagsClass->getStaticFieldValue(field);
+}
+
+static bool isSavePromiseJSInvocationStackEnabled() {
+  static bool savePromiseJSInvocationStack = getFeatureFlagBoolValue("traceTurboModulePromiseRejections");
+  return savePromiseJSInvocationStack;
+}
```

Comment:
Name this method after the feature flag so we don't introduce different terminology.